CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.0.4](https://github.com/rhobs/monitoring-stack-operator/commit/) (2021-11-25)
+
+
+### Features
+
+* deploy an Alertmanager instance for each monitoring stack ([e607afe](https://github.com/rhobs/monitoring-stack-operator/commit/e607afe23dd604845fad170d06a0cabb6aa1ad28))
+
+
+### Bug Fixes
+
+* ensure operator has no reconciliation errors ([5257706](https://github.com/rhobs/monitoring-stack-operator/commit/5257706d573c7e96adfd91b9d3e6565b168ab110))
+* query Prometheus through a dedicated service  ([58586e8](https://github.com/rhobs/monitoring-stack-operator/commit/58586e8c7cdfb077713077aa08149a9745b22d5f))
+* remove unnecessary debug log ([0bb0175](https://github.com/rhobs/monitoring-stack-operator/commit/0bb0175846af43e3025f805c936d662058cfb557))
```

Comment:
nit: this is insignificant but not worth waiting another 15m :)

Reply:
Yeah good point, I'll remove it